radar-upload-backend/src/main/java/org/radarbase/upload/resource/RecordResource.kt

```diff
@@ -25,8 +25,8 @@ class RecordResource {
     @Context
     lateinit var recordRepository: RecordRepository
 
-    @Context
-    lateinit var auth: Auth
+//    @Context
+//    lateinit var auth: Auth
```

Comment:
I would think that authentication is needed almost anywhere...

Reply:
True. But the `Auth` can only initialized on a request, but not when a resource is initialized.

Reply:
IIRC, it uses a proxy in that case, so the object is still instantiated per request.

Reply:
https://jersey.github.io/documentation/latest/jaxrs-resources.html#d0e2806